css/src/components/message.scss

```diff
@@ -72,3 +72,29 @@ $message-content-padding-sm: 0.875rem !default;
 		}
 	}
 }
+
+.loading-message {
```

Comment:
Do we need this? Can this be achieved with only .message, .progress-bar, and a few helpers?